test/pkcs7.test.js

```diff
@@ -0,0 +1,62 @@
+/* eslint-disable object-shorthand */
+
+import {pad, unpad} from '../src/pkcs7.js';
```

Comment:
This whole file was moved to follow the standard test naming format and converted to `QUnit` I think that its small enough to see that everything is still the same though.